### PR TITLE
Add order process modal

### PR DESCRIPTION
**Pull request by lgalis**
https://projects.engineering.redhat.com/browse/SSP-1710

Screenshot from 2020-07-31 15-16-48

Screenshot from 2020-07-31 15-16-21

Screenshot from 2020-07-31 15-16-38

### CODE REVIEW

File: src/forms/create-order-process.schema.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import componentTypes from '@data-driven-forms/react-form-renderer/dist/cjs/component-types';
+import validatorTypes from '@data-driven-forms/react-form-renderer/dist/cjs/validator-types';
+
+import formsMessages from '../messages/forms.messages';
+import labelMessages from '../messages/labels.messages';
+import debouncedValidatorName from './name-async-validator';
+
+/**
+ * Creates a data-driven-form schema for adding/editing portfolio
+ * @param {bool} newRecord sets the variant of portfolio form
```
Comment (Hyperkid123) [inline src/forms/create-order-process.schema.js:10]: the JS doc is not documenting this particular function

File: src/redux/actions/order-process-actions.js [hunk 2]
```diff
@@ -21,6 +25,20 @@ export const fetchOrderProcesses = (pagination) => (dispatch, getState) => {
...
 };
 
+export const addOrderProcess = (processData) => ({
+  type: ActionTypes.ADD_ORDER_PROCESS,
+  payload: OrderProcessHelper.addOrderProcess(processData),
+  meta: {
+    notifications: {
+      fulfilled: {
+        variant: 'success',
+        title: 'Success adding order process',
```
Comment (Hyperkid123) [inline src/redux/actions/order-process-actions.js:35]: notification is not translated

File: src/smart-components/order-process/order-processes.js [hunk 3]
```diff
@@ -129,11 +135,43 @@ const OrderProcesses = () => {
     );
   };
 
+  const routes = () => (
+    <Fragment>
```
Comment (Hyperkid123) [inline src/smart-components/order-process/order-processes.js:139]: The fragment node is not necessary.

**Comment by lgalis** [comment]
@Hyperkid123 - feedback addressed

**Pull request merged**